widget/table_test.go

```diff
@@ -117,6 +117,25 @@ func TestTable_Hovered(t *testing.T) {
 	defer w.Close()
 	w.Resize(fyne.NewSize(180, 180))
 
+	test.MoveMouse(w.Canvas(), fyne.NewPos(35, 100))
+
+	assert.Equal(t, -1, table.hoveredColumn)
+	assert.Equal(t, -1, table.hoveredRow)
+
+	test.AssertImageMatches(t, "table/nohovered.png", w.Canvas().Capture())
+
+	table = NewTable(
+		func() (int, int) { return 3, 5 },
+		func() fyne.CanvasObject {
+			return NewLabel("placeholder")
+		},
+		func(row, col int, c fyne.CanvasObject) {
+			text := fmt.Sprintf("Cell %d, %d", row, col)
+			c.(*Label).SetText(text)
```

Comment:
nit: could be one line
```suggestion
			c.(*Label).SetText(fmt.Sprintf("Cell %d, %d", row, col))
```